Dockerfile

```diff
@@ -86,13 +90,10 @@ FROM minimal-system as builder-production
 RUN apt-get update && \
     apt-get -y install --no-install-recommends \
         curl \
-        pkg-config \
```

Comment:
Don't we need pkg-config anymore?

Reply:
It's already being installed in the base layer, so removing it from here.

https://github.com/openedx/edx-platform/blob/ca2210259c98fadc7f848c0c6d1e9e1f3ae0c8b8/Dockerfile#L64